projects/guava/UrlEscapersFuzzer.java

```diff
@@ -0,0 +1,88 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import com.google.common.base.Charsets;
+import com.google.common.escape.Escaper;
+import com.google.common.net.UrlEscapers;
+import java.lang.IllegalArgumentException;
+import java.math.BigInteger;
+import java.net.URLDecoder;
+
+public class UrlEscapersFuzzer {
+
+	/*
+	 * these constants are private members copy-pasted from
+	 * com.google.common.net.UrlEscapers
+	 */
+	static final String URL_FORM_PARAMETER_OTHER_SAFE_CHARS = "-_.*";
+	static final String URL_PATH_OTHER_SAFE_CHARS_LACKING_PLUS =
+      "-._~" // Unreserved characters.
+          + "!$'()*,;&=" // The subdelim characters (excluding '+').
+          + "@:"; // The gendelim characters permitted in paths.
+
+    private static String toHex(String arg) {
+		return String.format("%x", new BigInteger(1, arg.getBytes(Charsets.UTF_8)));
+	}
+
+	private static void rejectSample(String original, String result, String encoded, String percentEncoded) {
+		System.out.println("sample: " + toHex(original));
+		System.out.println("encoded: " + encoded);
+		System.out.println("percentEncoded: " + percentEncoded);
+		System.out.println("result: " + toHex(result));
+		throw new RuntimeException();
```

Comment:
From what I understand, if the fuzzer goes here, it would be considered as findings. If it's the case, you should throw a Jazzer exception like `FuzzerSecurityIssueMedium`. You can read more about it in the doc https://codeintelligencetesting.github.io/jazzer-api/com/code_intelligence/jazzer/api/package-summary.html

Also, `println` is probably not needed, Jazzer will take care of it.